Core/src/main/java/com/plotsquared/core/util/SchematicHandler.java

```diff
@@ -771,6 +772,10 @@ public void run() {
                                     }
                                     BlockVector2 pt = BlockVector2.at(currentX, currentZ);
                                     BiomeType biome = aabb.getWorld().getBiome(pt);
+                                    // Replace empty biome with plains
+                                    if(biome == null) {
+                                        biome = BiomeTypes.PLAINS;
```

Comment:
Unsure if we should default to the plotworld's biome (as set in worlds.yml) rather than forcing plains

Reply:
Be a deal